resources/views/members/searchSkills.blade.php

```diff
@@ -0,0 +1,73 @@
+@extends('master')
+
+@section('title')
+Leden zoeken op vaardigheden & interesses
+@endsection
+
+@section('content')
+
+<h1>Leden zoeken op vaardigheden & interesses</h1>
+
+<hr/>
+
+@include('errors.list')
+
+<form class="form-inline" method="get" action="#">
+
+	<div class="form-group col-md-6">
+		<label for="skills">Vaardigheden: </label>
+		<select class="form-control" style="width: 100%;" name="skills[]" id="skills" multiple>
+			@foreach ($all_skills as $id => $skill)
+			<option value="{{$id}}" @if (in_array($id, $skills)) selected @endif>{{$skill}}</option>
+			@endforeach
+		</select>
+    </div>
+    
+    <div class="form-group col-md-4">
+        <label class="radio-inline">
```

Comment:
Can you name this "any" that is more descriptive

Reply:
Yeah good one

Reply:
I don't see it on mobile, but I trust you on it

Reply:
I don't see it on mobile, but I trust you on it